definitions/features/downstream.rb

```diff
@@ -74,6 +76,14 @@ def rh_repos(sat_version)
     rh_repos
   end
 
+  def enable_ansible_repo(sat_version, rh_version_major)
```

Comment:
@upadhyeammit just a nitpick: this method has misleading name as it does not enable anything but rather returns the repo so IMO it should be `ansible_repo`.

Reply:
changed method name, @mbacovsky thank you!